api/policy/policy.go

```diff
@@ -0,0 +1,163 @@
+package policy
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/api/header"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/CircleCI-Public/circleci-cli/version"
+)
+
+// Client communicates with the CircleCI policy-service to ask questions
+// about policies. It satisfies policy.ClientInterface.
+type Client struct {
+	serverUrl string
+	client    *http.Client
+}
+
+type httpError struct {
+	Error   string                 `json:"error"`
+	Context map[string]interface{} `json:"context,omitempty"`
+}
+
+func (c Client) ListPolicies(ownerID string, activeFilter *bool) (interface{}, error) {
```

Comment:
The goal of returning the empty interface was that we essentially just want to passthrough the server response to the CLI. 
If we wrote reponse types and the server added fields, we would need to come back to the CLI and add them there. Since we aren't interacting with the values other than pretty printing them to the user, we chose to parse as an empty interface.

Do you see any drawbacks?

Reply:
The only reason we parse into a value instead of streaming the response directly to stdout is because we wanted to pretty print it :/, I wasn't aware of a good way to pretty print a json stream

Reply:
If I recall correctly, as long as the struct has json annotations for it's fields, simply doing a `fmt.Printf("%+v", struct)` should print the JSON.

I think one of the biggest benefits of using a typed language is for developers to easily see the data structures they're working with in their code. And even though it'll be a little bit of maintenance, it'll make it easier for developers to see what they're working with. Ideally, the API itself should be able to generate these clients rather than having it maintained by us.

I'm not strongly opinionated on this, I think it's fine the way it is since you are just streaming the output but my preference would be types.